dotnet/InkRecognition/wpf-app/src/NoteTaker.csproj

```diff
@@ -78,7 +78,6 @@
     <Reference Include="System.Runtime.Serialization" />
     <Reference Include="System.Runtime.WindowsRuntime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>C:\Windows\Microsoft.NET\Framework\v4.0.30319\System.Runtime.WindowsRuntime.dll</HintPath>
```

Comment:
Not needed?

Reply:
I tried some fresh machine and this line is not needed. VS can find a WindowsRuntime at somewhere under C:\Windows\Microsoft.Net\assembly

Reply:
Got it!